packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -172,6 +173,20 @@ abstract class ScrollView extends StatelessWidget {
   /// {@macro flutter.rendering.viewport.cacheExtent}
   final double cacheExtent;
 
+  /// The number of children which will contribute semantic information.
```

Comment:
nit: "that will"? I think when you say "which" you refer to "the number", not "children".